accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -245,11 +241,7 @@ Windows-specific:
     - `AddEventHandler(Object, Delegate)`
     - `RemoveEventHandler(Object, Delegate)`
 * **ComEventsHelper**
-    - `Combine(Object, Guid, Int32, Delegate)`
-    - `Remove(Object, Guid, Int32, Delegate)`
```

Comment:
all methods of this type are Windows-specific so we should most probably make the entire type as such

https://github.com/dotnet/runtime/blob/e7204f5d6fcaca5e097ec854b3be6055229fc442/src/libraries/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs#L224-L228